packages/components/src/atoms/PaneCloseIcon/PaneCloseIcon.tsx

```diff
@@ -29,14 +24,13 @@ export const PaneCloseIcon: React.FC<PaneCloseIconType> = props => {
       style={{...containerStyle}}
       onClick={onClick}
     >
-      <StyledDoubleLeftOutlined
-        className={iconClassName}
-        style={{...iconStyle}}
-      />
+      <StyledDoubleLeftOutlined className={iconClassName} style={{...iconStyle}} />
     </StyledPaneCloseIconContainer>
   );
 };
 
+export default PaneCloseIcon;
```

Comment:
No need for action, more to inform, but I noticed @monojack does something pretty cool in our codebase. For certain components he exports it as both named and default. That way you have the benefits of easier named imports, yet the ability to lazily import when needed.